__tests__/routes/submissions.test.js

```diff
@@ -60,7 +60,7 @@ describe('Testing childSubmissionRouter endpoints', () => {
         storyId: 2,
         episodeId: 1,
       };
-      childSubmissionsModel.getSubmissionByChildId.mockResolvedValue(undefined);
+      // childSubmissionsModel.getSubmissionByChildId.mockResolvedValue(undefined);
```

Comment:
Let's remove this commented out code

Reply:
yes it can and it was removed in the latest commit